Automattic/wp-calypso
# Themes: Adding alert on themes that need to be updated

#### Changes proposed in this Pull Request

* Adds an alert in the outdated themes to allow the users to update it. The number in the badge in the sidebar menu would inform the number of Themes that need to be updated, but the user could not take action to update it.

![2022-05-24 16 26 11](https://user-images.githubusercontent.com/1234758/170074145-16483136-3cad-453e-9c8f-ae9347e3d826.gif)

#### Testing instructions

* Go to the `Themes` under `Appearance`.
* Click on `Install Theme` and upload an outdated theme, for example, https://downloads.wordpress.org/theme/storefront.4.0.0.zip
* Then, you should see a badge with a number in the `Themes` menu.
* Go to the `Themes` page again and select the `My Themes` tab, which should also contain the badge with the number.
* You'll see the outdated themes with alerts, indicating that they need to be updated.
* Click on the button `Update Theme`, and the numbers in the badges should decrease or disappear in case no theme requires an update

Closes #62822
